File: api/Dtos/Plan/PlanEditReq.cs
using api.Entities;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using api.Dtos.PlanCategory;

namespace api.Dtos.Plan
{
    public class PlanEditReq
    {
        [Required, MinLength(3)]
        public string? Name { get; set; }
        public string? Description { get; set; }
        public int? PlanCategoryId { get; set; }
        public int? PlanTypeId { get; set; }
        public int SetupFee { get; set; } = 0;
        public int Price { get; set; } = 0;
    }
}

File: web/src/dtos/Plan/PlanEditReq.ts
export default class PlanEditReq {
  name?: string = "";
  description?: string = "";
  planCategoryId?: string = "";
  planTypeId?: string = "";
  duration?: number = 0;
  timeUnitId?: string = "";
  setupFee?: number = 0;
  price?: number = 0;
}
